README.md

```diff
@@ -110,15 +127,67 @@ If you've set up `lnmuxd` correctly, output similar to what is shown below is ex
 
 ## Invoice lifecycle
 
-![](invoice_lifecycle.png)
+![](images/invoice_lifecycle.png)
 
 Note that only the states `accepted`, `settle requested` and `settled` are published to callers of the `SubscribeSingleInvoice` rpc. Lnmux isn't aware of open invoices because it is stateless.
 
 The transition from `settle requested` to `settled` can be made more robust in the future. This transition is happening already, but not backed by an actual final settle event from lnd. See https://github.com/lightningnetwork/lnd/issues/6208.
 
+## Docker Instructions
+
+We provide a Dockerfile to launch lnmux using docker.
+
+This Dockerfile will build 2 different images: one for debian and the other one for alpine.
+
+To build the lnmux images, navigate to the lnmux repository and simply do:
+
+```bash
+docker build .
+```
+
+**However**, we don't provide any configuration file in the Dockerfile (and don't do the migration either).
+You have various options:
+
+1) Use a `docker-compose.yml` to mount a volume to provide your configuration file, and then override the command instructions.
```

Comment:
The original issue didn't mention docker-compose, just a command line how to launch lnmux in docker. I am not sure about the docker-compose addition. It still requires lnd nodes that run on the host, where I'd expect docker-compose to be a completely self-contained setup including lnd nodes and bitcoind. The way you defined it currently, it's just lnmux + postgres. And a path that needs to be changed.

I think it is better to limit ourselves to a `docker run` command line that mounts a config file into the pre-built image.